stock_reserve/models/product.py

```diff
@@ -46,6 +46,20 @@ def _compute_reservation_count(self):
             reservations = self.env["stock.reservation"].search(domain)
             product.reservation_count = sum(reservations.mapped("product_qty"))
 
+    def _get_domain_locations_new(
+        self, location_ids, company_id=False, compute_child=True
+    ):
+        """Add location reservation for avoiding reservations to be substracted on
+        the forecasted quantity."""
```

Comment:
Remember how docstrings are put according the PEP:

```suggestion
        """Add location reservation for avoiding reservations to be substracted on
        the forecasted quantity.
        """
```